src/gui/properties/speedwidget.cpp

```diff
@@ -48,8 +48,9 @@ ComboBoxMenuButton::ComboBoxMenuButton(QWidget *parent, QMenu *menu)
 
 void ComboBoxMenuButton::showPopup()
 {
-    QPoint p = mapToGlobal(QPoint(0, height()));
-    m_menu->exec(p);
+    const QPoint p = mapToGlobal(QPoint(0, height()));
+    m_menu->popup(p);
```

Comment:
You omit `setAttribute(Qt::WA_DeleteOnClose);` before `popup()` in some cases. Is it intentional?

Reply:
In this case the `m_menu` real lifetime is tied to the class so we don't need to delete it.
I have checked all instances of QMenu visible in this PR.